recipes/ericscript/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: ericscript
+  version: 0.5.5
+
+source:
+  fn: ericscript-0.5.5.tar.bz2
+  url: https://sourceforge.net/projects/ericscript/files/ericscript-0.5.5.tar.bz2 
```

Comment:
Please add a [hash](http://conda.pydata.org/docs/building/meta-yaml.html#source-from-tarball) for the archive file, so conda build will check its integrity. An md5 hash is fine, but sha1 and sha256 are also supported.